sensor/extrinsic_reflector_based_calibrator/src/extrinsic_reflector_based_calibrator.cpp

```diff
@@ -1280,6 +1324,112 @@ void ExtrinsicReflectorBasedCalibrator::calibrateSensors()
       "The calibrated poses differ considerably with the initial calibration. This may be either a "
       "fault of the algorithm or a bad calibration initialization");
   }
+
+  // for plotter and rviz display
+  output_calibration_distance_error = static_cast<float>(calibrated_2d_distance_error);
+  output_calibration_yaw_error = static_cast<float>(calibrated_2d_yaw_error);
+}
+
+void ExtrinsicReflectorBasedCalibrator::crossValEvaluation(
+  pcl::PointCloud<PointType>::Ptr lidar_points_pcs,
+  pcl::PointCloud<PointType>::Ptr radar_points_rcs)
+{
+  // Evaluate the cross validation metric using SVD
+  double cv_distance_error = 0.0;
+  double cv_yaw_error = 0.0;
+  auto get_yaw_error = [](const Eigen::Vector3d & v1, const Eigen::Vector3d & v2) -> double {
+    return std::abs(std::acos(v1.dot(v2) / (v1.norm() * v2.norm())));
+  };
+
+  if (converged_tracks_.size() > 1) {
```

Comment:
In more software oriented companies one of the metrics people try to minimize is code complexity.
One of its elements is correlated with the number of indentations.
In this case, this indentation can be removed by changing the condition to <=1 and addressing that case as an "easy case" and returning early

Reply:
Thanks! I got it. However, if I write it as if (converged_tracks_.size() <= 1), then I need to write the RCLCPP_INFO and the output_error twice. Is is still better?

Reply:
Sorry, I don't understand why you would need to output it twice.
You could just early return

Reply:
For instance: 
```
if (converged_tracks_.size() > 1) {
  //We still need to assign the value to output and also output the log.
  RCLCPP_INFO(...)
  output_crossval_distance_error = static_cast<float>(crossval_distance_error);
  output_crossval_yaw_error = static_cast<float>(crossval_yaw_error);
  return;
}
// do crossval
RCLCPP_INFO(...)
output_crossval_distance_error = static_cast<float>(crossval_distance_error);
output_crossval_yaw_error = static_cast<float>(crossval_yaw_error);
```

Of course, I can also rewrite it like this:
output_crossval_distance_error = 0;
output_crossval_yaw_error = 0;
if (converged_tracks_.size() > 1) {
  //But we still need to output the log here, or you mean you don't want to output the log if the track size is less than 2.
  RCLCPP_INFO(...)
  return;
}
// do crossval
RCLCPP_INFO(...)
```

Reply:
Both are fine, at least I don't feel any resistance to any of the options.
Just that the log may be different at does not need to use the variables.

Reply:
commit https://github.com/tier4/CalibrationTools/pull/145/commits/d3f0c4ef8783569b7aa890cdd45f89544a461663 fix this problem.